ops/terraform/services/server/modules/bfd_server_asg/main.tf

```
@@ -246,33 +251,34 @@ resource "aws_autoscaling_policy" "filtered_networkin_low_scaling" {
     # and the .0 precision modifier to ensure that Terraform's formatter does not pad the decimal
     # part with 0s
     metric_interval_upper_bound = format("%.0e", -300 * 1000000) # 300 megabytes
-    scaling_adjustment          = -(length(var.env_config.azs) * 3)
+    scaling_adjustment          = length(var.env_config.azs)
   }
 
   step_adjustment {
     metric_interval_lower_bound = format("%.0e", -300 * 1000000) # 300 megabytes
     metric_interval_upper_bound = format("%.0e", -200 * 1000000) # 200 megabytes
-    scaling_adjustment          = -(length(var.env_config.azs) * 2)
+    scaling_adjustment          = length(var.env_config.azs) * 2
   }
 
   step_adjustment {
     metric_interval_lower_bound = format("%.0e", -200 * 1000000) # 200 megabytes
     metric_interval_upper_bound = 0                              # 0 megabytes
-    scaling_adjustment          = -length(var.env_config.azs)
+    scaling_adjustment          = length(var.env_config.azs) * 3
   }
 }
 
 resource "aws_cloudwatch_metric_alarm" "filtered_networkin_high" {
   alarm_name          = "bfd-${var.role}-${local.env}-networkin-high"
-  comparison_operator = "GreaterThanThreshold"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
   datapoints_to_alarm = 1
   evaluation_periods  = 1
-  threshold           = 100 * 1000000 # 100 megabytes
+  threshold           = 1
   treat_missing_data  = "ignore"
```

```suggestion
  treat_missing_data  = "notBreaching"
```

Missed this one! The Alarm's metric is sparse, and if there is no value for said metric during the Alarm's evaluation period that means that there should be no change in scaling capacity. So, the Alarm's state should be set to `OK`.